webpack.config.babel.js

```diff
@@ -93,17 +101,50 @@ const webPackConfig = {
     path: path.resolve(__dirname, 'dist'),
     publicPath: '/',
   },
+  devServer: {
+    hot: true,
+    index: '',
+    proxy: [
+      {
+        // Proxy all server-served routes:
+        context: ['/img', '/api', '/auth', '/socket.io', 'sockjs-node'],
+        target: 'http://localhost:9090',
```

Comment:
should this be moved to some var?

Reply:
the target?

Reply:
I think it's clear like this - it is not used twice